.github/workflows/mesheryctl-e2e.yaml

```
@@ -0,0 +1,140 @@
+name: Meshery End-to-End Tests with mesheryctl
+on:
+  workflow_dispatch:
+    inputs:
+      profile_name:
+        description: "performance profile to use"
+        required: true
+      profile_filename:
+        description: "test configuration file"
+        required: false
+      patternfile_url:
+        description: "URL of the pattern file"
+        required: false
+      patternfile_name:
+        description: "name of the pattern file"
+        required: false
+  # scheduled to run at the 43rd minute of every 12th hour
+  schedule:
+    - cron: '43 */12 * * *'
```

```suggestion
    - cron: '43 */24 * * *'
```

Out of the 2,000 available a month, do you know how many build minutes this eats up? Let's start with once-a-day.